ipn/ipnlocal/local.go

```diff
@@ -199,6 +199,7 @@ type LocalBackend struct {
 	// ServeConfig fields. (also guarded by mu)
 	lastServeConfJSON mem.RO              // last JSON that was parsed into serveConfig
 	serveConfig       ipn.ServeConfigView // or !Valid if none
+	serveListeners    map[uint16]map[string]net.Listener
```

Comment:
always document what map keys/values are, especially for nest maps, like:

     serveListeners    map[uint16]map[string]net.Listener // foo => barID => listener